src/runtime/entitlements-manager-test.js

```diff
@@ -64,6 +66,7 @@ describes.realWin('EntitlementsManager', {}, env => {
     sandbox.stub(deps, 'storage', () => storage);
     sandbox.stub(deps, 'pageConfig', () => pageConfig);
     sandbox.stub(deps, 'config', () => config);
+    sandbox.stub(deps, 'eventManager', () => eventManager);
     const analyticsService = new AnalyticsService(deps);
```

Comment:
Line 70-72 can be removed since EntitlementsManager no longer interacts with analytics Service.